src/doc/src/reference/unstable.md

```diff
@@ -1170,25 +1169,6 @@ cargo logout -Z credential-process
 [crates.io]: https://crates.io/
 [config file]: config.md
 
-### rust-version
```

Comment:
Can you add a section at the bottom to indicate that this is now stable (see the other entries for examples).

Reply:
Done.